mixins/list.go

```diff
@@ -231,10 +244,16 @@ func (l *List) SetScrollOffset(scrollOffset int) {
 	}
 	s := l.outer.Size().Contract(l.outer.Padding())
 	if l.orientation.Horizontal() {
+		if l.scrollRound && l.itemSize.W != 0 {
+			scrollOffset = scrollOffset + scrollOffset%l.itemSize.W
+		}
 		maxScroll := math.Max(l.itemSize.W*l.itemCount-s.W, 0)
 		scrollOffset = math.Clamp(scrollOffset, 0, maxScroll)
 		l.scrollBar.SetScrollPosition(scrollOffset, scrollOffset+s.W)
 	} else {
+		if l.scrollRound && l.itemSize.H != 0 {
+			scrollOffset = scrollOffset + scrollOffset%l.itemSize.H
```

Comment:
Similar thing here - you should use `-=` because the remainder needs to be subtracted, not added.